src/Updater.php

```diff
@@ -20,6 +20,12 @@ class Updater implements UpdaterInterface {
 
   use StringTranslationTrait;
 
+  const CONFIG_NOT_FOUND = 0;
+  const CONFIG_ALREADY_APPLIED = 1;
+  const CONFIG_NOT_EXPECTED = 2;
+  const CONFIG_APPLIED_SUCCESSFULLY = 3;
+  const MODULES_FOUND = 4;
```

Comment:
I would leave out `module` related states since we don't have any log messages related to that right now. And maybe we should add them with PR where we are adding expected modules functionality.